### PR TITLE
Add boogie woogie, matrix, whale

### PR DESCRIPTION
**Pull request by duizendnegen**

### CODE REVIEW

**Comment by osresearch** [comment]
This looks like it should merge in with no issues! I'll add them into tonight's mix.

File: index.html [hunk 1]
```diff
@@ -65,6 +65,18 @@ <h1>Lite Festival</h1>
 			<img src="images/clock.jpg" alt="Glowing Seven Segment Clock" class="sketch" />
 			<p class="artist-name">Trammell Hudson</a> </p>
 		</div>
+		<div class="flex-container-col">
+			<img src="images/boogiewoogie.png" alt="Metro Boogie Woogie" class="sketch" />
+			<p class="artist-name">Esther &amp; Pepijn</p>
```
Comment (osresearch) [inline index.html:70]: do you want a link to your website?

Comment (duizendnegen) [reply index.html]: Done :)

**Comment by duizendnegen** [comment]
Thanks! We'll visit Tuesday eve to check it out ❤️

**Pull request merged**